conf/messages.cy

```diff
@@ -0,0 +1,3935 @@
+#===================================================
+# TRANSLATION NOT REQUIRED = START
+#===================================================
+
```

Comment:
@nubz @AndyHWChung copied the current messages in just for testing purposes - should I take this out?

Reply:
you could maybe take only the non-agent start page content and the onboarding check your details page content  and replace the english content with random crap. Then we can see clearly if we toggle the language on the start page whether or not it works.

Reply:
I tested it locally and the crap works

Reply:
I'm going to take all this out and switch it off

Reply:
for the purposes of QA'ing it, it'd be easier to test if you leave at least a couple of pages with some random translations - when we eventually start getting translations through we can then replace the crap with the actual thing

Reply:
just translated the account home page